cypress.config.js

```diff
@@ -1,8 +1,34 @@
-const { defineConfig } = require('cypress')
+const { defineConfig } = require('cypress');
+const faker = require('faker'); 
 
 module.exports = defineConfig({
   e2e: {
+    baseUrl: 'https://demoqa.com/automation-practice-form', 
+    viewportHeight: 1080,
+    viewportWidth: 1920,
     setupNodeEvents(on, config) {
+      on('task', {
+        generateUser() {
+          genders = ['Male', 'Female', 'Other'];  
+          subjects = ['Chemistry', 'English', 'Math', 'Computer Science']
+          hobbies = ['Sports', 'Reading', 'Music'];
```

Comment:
Declare the arrays 'genders', 'subjects', and 'hobbies' as constants outside of the 'generateUser' function to avoid redeclaring them each time the function runs.

Reply:
done